### PR TITLE
Hl 1361 missing full application

### PR DESCRIPTION
**Pull request by rikuke**
## Description :sparkles:
[HL-1361](https://helsinkisolutionoffice.atlassian.net/browse/HL-1361?atlOrigin=eyJpIjoiY2Q5NDcyMzg0MmU5NDM5ZTliNzdhZTJkOGEzMTQxMjQiLCJwIjoiaiJ9)
This time ensure, that the full application attachment is included in the pre fetch query, where it was not previously included.
## Additional notes :spiral_notepad:

[HL-1361]: https://helsinkisolutionoffice.atlassian.net/browse/HL-1361?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: backend/benefit/applications/services/ahjo_payload.py [hunk 1]
```diff
@@ -272,7 +272,9 @@ def prepare_update_application_payload(
     """Prepare the payload that is sent to Ahjo when an application is updated, \
           in this case it only contains a Records dict"""
     if not pdf_summary.ahjo_version_series_id:
-        raise ValueError("Attachment must have an ahjo_version_series_id for update.")
+        raise ValueError(
+            f"Attachment for {application.application_number} must have a ahjo_version_series_id for update."
```
Comment (rikuke) [inline backend/benefit/applications/services/ahjo_payload.py:276]: While not related to this issue, having this helps debug update request to hyte more efficiently.

File: backend/benefit/applications/tests/test_ahjo_integration.py [hunk 2]
```diff
@@ -716,6 +716,30 @@ def test_generate_ahjo_secret_decision_text_xml(decided_application):
         os.remove(attachment.attachment_file.path)
 
 
+@pytest.mark.django_db
+def test_get_applications_for_ahjo_update(
```
Comment (rikuke) [inline backend/benefit/applications/tests/test_ahjo_integration.py:720]: Test that update request gets the right amount of applications.

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr3089.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr3089.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr3089.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-handler-pr3089.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr3089.dev.hel.ninja :laughing::tada::tada::tada:

**Review by sirtawast — APPROVED** [review]

**Pull request merged**